binder/postBuild

```diff
@@ -0,0 +1,4 @@
+jupyter labextension install @jupyter-widgets/jupyterlab-manager --no-build
+jupyter labextension install ipysheet --no-build
+jupyter labextension install jupyterlab-plotly --no-build
+jupyter lab build --dev-build=False --minimize=False
```

Comment:
I just realized that to make the `MissionViewer.display()` work in Jupyter Lab you have to install `jupyter labextension install plotlywidget` since it uses a plotly `FigureWidget`. Ashame it is not covered by `jupyterlab-plotly`.

Reply:
Thanks for the advice.
Done in cad824984a6492882318d4d948f45a7e723b4e93.